package.json

```diff
@@ -18,5 +18,8 @@
   "homepage": "https://github.com/jkirkwin/CourseChart#readme",
   "dependencies": {
     "express": "^4.17.1"
+  },
+  "devDependencies": {
```

Comment:
Should this be a dev dependency or a normal one? [This post](https://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies) talks about the difference pretty concisely - it sounds to me like it should be a prod dependency bc we'll need postgres to run the app